js/ad_manager_vast.js

```diff
@@ -372,16 +367,24 @@ OO.Ads.manager(function(_, $) {
       var linearAd = ad.data.linear;
       if (linearAd && linearAd.mediaFiles) {
         var vastStreams = linearAd.mediaFiles;
-        streams["webm"] = this._extractStreamForType(linearAd.mediaFiles, "webm");
-        if (ad.streamUrl == null) {
-          streams["mp4"] = this._extractStreamForType(linearAd.mediaFiles, "mp4");
-          OO.log("extract m3u8 stream here"); // TODO
+        var videoEncodingsSupported = OO.VIDEO.ENCODING;
+        var streamData = null;
```

Comment:
I would rather leave it outside for clarity sake, if you are strongly in the opinion that it should be outside I will move it.